client/src/pages/Home.css

```
@@ -9,3 +9,46 @@
 .message:hover {
 	background-color: yellow;
 }
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(214, 227, 243);
+  align-items: center;
+  height: 100%;
+  border-radius: 12px;
+  padding: 100px 100px 100px 100px;
+  width: 70%;
+  margin-left: 10em;
+  box-shadow: 3px 3px rgb(143, 139, 139), -1em 0 .4em rgb(211, 211, 198);
```

```suggestion
  box-shadow: 3px 3px 10px rgb(143, 139, 139);
```
`-1em` does not work for the second shadow because it's a negative number, so let's remove it. We can also add a blur radius of `10px` to make the shadow more blurred

---

I removed the shadow box